BotModules/Pager/Users.json

```diff
@@ -0,0 +1,9 @@
+{
+    "352255034022690828": "DOYA_KNOME",
+    "135928222696210433":"XTREME_NERD",
+    "121423130907377664":"BEASTY_ROGUE",
+    "fuuuk boi":"BWERICK",
+    "juuuk boi2":"LX",
+    "FUUK boi3":"NARCUGAR",
+    "276760223853576192":"JON"
```

Comment:
This info is stored in the DB, I suggest we pull from there to avoid having this publicly available

Right now the schema is 

- Members
    - _id: (discID)
    - steamID
    - DiscordAlias


I'll make a branch with some functions you can call and link them once I'm finished

Reply:
https://github.com/PhilipBeckwith/Discord-Bots/commit/7b67423479c3de61053d258818703f312e34dcf0 

I've added a method to the new Mongo class that you can use to get all of the fields shown above. It will return a json object that you can use to get specific fields. 

The next step would be to add the phone numbers to the member object in the database so you can use something like

member.phone

I can take care of adding the phone numbers

Reply:
Numbers are now part of every Member object. Object structure excluding games array is below
![image](https://user-images.githubusercontent.com/11918599/90320512-d11ed000-def6-11ea-9b0b-c31673d67400.png)

Reply:
Mongo wasn't working. So I didn't go with this option.